datadog_lambda/wrapper.py

```diff
@@ -40,19 +47,29 @@ class _LambdaDecorator(object):
 
     def __init__(self, func):
         self.func = func
-        self.flush_to_log = os.environ.get('DD_FLUSH_TO_LOG', '').lower() == 'true'
-        self.logs_injection = os.environ.get('DD_LOGS_INJECTION', '').lower() == 'true'
+        self.flush_to_log = os.environ.get("DD_FLUSH_TO_LOG", "").lower() == "true"
+        self.logs_injection = os.environ.get("DD_LOGS_INJECTION", "").lower() == "true"
 
         # Inject trace correlation ids to logs
         if self.logs_injection:
             inject_correlation_ids()
 
         # Patch HTTP clients to propagate Datadog trace context
         patch_all()
-        logger.debug('datadog_lambda_wrapper initialized')
+        logger.debug("datadog_lambda_wrapper initialized")
 
     def _before(self, event, context):
+        global cold_start_request_id
+        # Assign this request ID as the cold start if there is no value yet
+        if cold_start_request_id is None:
+            cold_start_request_id = context.aws_request_id
+
         try:
+            lambda_metric(
```

Comment:
Although this is a oneliner, I would make it a function in metric.py, to keep the wrapper focus on code flow rather than details.